src/core.jl

```diff
@@ -1,22 +1,44 @@
 
 import StatsBase.fit
 import StatsBase.predict
+import StatsBase.span
```

Comment:
> If you need to use using StatsBase, you might as well remove the imports. You will also need to do (StatsBase.fit( ), StatsBase.predict() etc.) to all the methods below.

Hmm, for me it sounds easier to write `import StatsBase: fit, predict, span` once rather than add `StatsBase.` to all such methods in core.jl. My I know why you prefer the latter? 

> Move the line using StatsBase into the NaiveBayes.jl file.

Oh, please don't do it. I'm heavy REPL user and build my workflow around 2 simple concepts: 

 * create single file - normally called `core.jl` - that contains or includes all the code of a module;
 * `include("core.jl")` into a module code itself, i.e. `NaiveBayes.jl`.

This allows me to load `core.jl` into REPL's Main and update functions incrementally, without the need to reload the target module and thus invalidate current variables. 

I assume it's not really important detail for you, but for me keeping all imports in `core.jl`  makes things much easier.